Corona-Warn-App/src/main/java/de/rki/coronawarnapp/datadonation/analytics/modules/registeredtest/TestResultDonor.kt

```diff
@@ -40,16 +40,22 @@ class TestResultDonor @Inject constructor(
         }
 
         val lastChangeCheckedRiskLevelTimestamp = testResultDonorSettings.mostRecentDateWithHighOrLowRiskLevel.value
-        if (lastChangeCheckedRiskLevelTimestamp == null) {
-            Timber.d("Skipping TestResultMetadata donation (lastChangeCheckedRiskLevelTimestamp is missing)")
-            return TestResultMetadataNoContribution
-        }
 
-        val daysSinceMostRecentDateAtRiskLevelAtTestRegistration =
+        val daysSinceMostRecentDateAtRiskLevelAtTestRegistration = if (lastChangeCheckedRiskLevelTimestamp == null) {
```

Comment:
The null case is covered by the method calculateDaysSinceMostRecentDateAtRiskLevelAtTestRegistration

Reply:
I was not aware of that :) . this is something that Max just updated the specs yesterday to have it

Reply:
Addressed in https://github.com/corona-warn-app/cwa-app-android/pull/2663/commits/e8ba3a0b5b5405a422fbdbb816f9557fee2e1aee